components/BottomMenu.js

```diff
@@ -97,39 +75,33 @@ function BottomMenu(props) {
     };
 
     const goToNearby = () => {
-        AsyncStorage.setItem("sideMenu", "mapView");
+        AsyncStorage.setItem("sideMenu", "mapView"); //FIXME: WHY?
```

Comment:
Yes I had answered, in the updated code, its for the NearbyInterest Component, so that it can know from what component the user was coming from (ex: bottom menu or side menu)

Reply:
It was clarified!